Adding Feat. Engineering Part of Bank Marketing

Nada-gh opened the pull request.

Nada-gh committed:
Deleted csv file for the test data

Nada-gh committed:
a little change to the code

imkolla commented:
use pd.get_dummies as in 
'''
pd.get_dummies(BankData[['job','marital','education','housing']])
'''
to transform all the categorical values into one hot encoded feature columns

Nada-gh committed:
adding some changes on the feature engineering part

-mapping categorical data with dummy variables
-normalizing continuous data
- mapping binary variables into (1/0)

Nada-gh committed:
Adding the final version of the pre-processed data

Nada-gh committed:
a little change on the day-interval variable

Khaledbrig committed:
adding the data modelling part of the project

Khaledbrig committed:
a slight change to the final saved data

Khaledbrig committed:
a small change for the pipeline setting

Nada-gh committed:
setting test

Nada-gh committed:
Adding the Feaure Selection file

Nada-gh committed:
Adding the updated version of Data Modeling file

Nada-gh committed:
A slight change in the code

Nada-gh committed:
a slight change on the PCA plot

Nada-gh committed:
a slight change on the PCA part
